packages/ui-core/core/src/lib/services/users/users.service.ts

```
@@ -1,81 +1,95 @@
-	 * @returns A Promise resolving to the IUser object.
+	 * @param relations - An array of relation names to include in the response.
+	 * @param includeEmployee - Whether to include employee details.
+	 * @returns A promise that resolves to the IUser object.
 	 */
-	getMe(relations: string[] = [], includeEmployee: boolean = false): Promise<IUser> {
-		return firstValueFrom(
-			this.http.get<IUser>(`${this.API_URL}/me`, {
-				params: toParams({ relations, includeEmployee })
-			})
-		);
+	async getMe(relations: string[] = [], includeEmployee: boolean = false): Promise<IUser> {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)